ui/imports/Themes/DarkTheme.qml

```diff
@@ -5,7 +5,7 @@ Theme {
     property color white: "#FFFFFF"
     property color white2: "#FCFCFC"
     property color black: "#000000"
-    property color grey: "#EEF2F5"
+    property color grey: "#141414"
```

Comment:
Is this intentional?

Reply:
Yes, likely due to changes made in https://github.com/status-im/nim-status-client/pull/698. I originally had all the changes in one PR, but split them up for clarity. Maybe I should have changed it in one PR haha.

Reply:
Have set it back, and will handle any needed changes in https://github.com/status-im/nim-status-client/pull/698.